server/dataInitialization/basicDemo.js

```diff
@@ -28,6 +28,18 @@ const userAccounts = {
 		},
 	},
 };
+*/
+
+const userAccounts = {
```

Comment:
Even though they have reduced permissions, do we still want default accounts in a config file/env variables?